pages/api/hasClaimed.ts

```diff
@@ -0,0 +1,11 @@
+import Redis from "ioredis";
+import { REDIS_URL } from "@/libs/constants";
+
+const client = new Redis(REDIS_URL);
+
+async function hasClaimed(address: string, network: string): Promise<boolean> {
+	const resp: string | null = await client.get(`${address}-${network}`);
```

Comment:
should we do it per twitter account here? to prevent people from just switching wallets and revalidating with same twitter?

Reply:
claim is now against address/network/assetId to match up with your PR - twitter account doesn't matter here basically its once per day per CENNZ account for each network

Reply:
okay sweet yeah think that should provide enough resistance